conform-pr/main.go

```diff
@@ -164,28 +188,38 @@ func checkSize(_ *githubactions.Action, projectFields map[string]graphql.Fields)
 	projects := maps.Keys(projectFields)
 	slices.Sort(projects)
 
+	var got []string
 	for _, project := range projects {
 		if size := projectFields[project]["Size"]; size != "" {
-			return fmt.Errorf(`PR for project %q has "Size" field set to value %q; it should be unset.`, project, size)
+			got = append(got, fmt.Sprintf("%q for project %q", size, project))
 		}
 	}
 
+	if got != nil {
+		return fmt.Errorf(`PR should have "Size" field unset, got %s.`, strings.Join(got, ", "))
+	}
+
 	return nil
 }
 
-// checkSprint checks that PR has a "Sprint" field set.
-func checkSprint(_ *githubactions.Action, projectFields map[string]graphql.Fields) error {
+// checkSprint checks that PR has a "Sprint" field set for non-community PRs.
```

Comment:
I don't understand the godoc comment. It says that the check is for non-community PRs, but it actually checks it for all the PRs, not only for non-community PRs.

Reply:
Done